src/build_scripts/buildlib.cprnc

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+from standard_script_setup import *
+from CIME.utils import run_bld_cmd_ensure_logging
+from CIME.case import Case
+from CIME.build import get_standard_cmake_args
+
+logger = logging.getLogger(__name__)
+
+def parse_command_line(args, description):
+###############################################################################
+    parser = argparse.ArgumentParser(
+        usage="""\n{0} [--debug]
+OR
+{0} --verbose
+OR
+{0} --help
+
+\033[1mEXAMPLES:\033[0m
+    \033[1;32m# Run \033[0m
+    > {0}
+""" .format (os.path.basename(args[0])),
+
+description=description,
+
+formatter_class=argparse.ArgumentDefaultsHelpFormatter
+)
+
+    CIME.utils.setup_standard_logging_options(parser)
+
+    parser.add_argument("buildroot",
+                        help="build path root")
+
+    parser.add_argument("installpath",
+                        help="install path ")
+
+    parser.add_argument("caseroot", nargs="?", default=os.getcwd(),
+                        help="Case directory to build")
+
+    args = CIME.utils.parse_args_and_handle_standard_logging_options(args, parser)
+
+    return args.buildroot, args.installpath, args.caseroot
+
+###############################################################################
+def buildlib(bldroot, installpath, case):
+###############################################################################
+    cimeroot = case.get_value("CIMEROOT")
+
+    # Generate macros and environment
+    run_bld_cmd_ensure_logging("{}/tools/configure --mpilib=mpi-serial --macros-format=CMake".format(cimeroot), logger, from_dir=bldroot)
+
+    cmake_args = get_standard_cmake_args(case, shared_lib=True)
+
+    cmake_cmd = "CIMEROOT={cimeroot} . .env_mach_specific.sh && NETCDF=$(dirname $(dirname $(which nf-config))) CIMEROOT={cimeroot} cmake {cmake_args} -DMPILIB=mpi-serial -DDEBUG=FALSE -C Macros.cmake {cimeroot}/tools/cprnc -DCMAKE_PREFIX_PATH={dest_path}".\
```

Comment:
Is nf-config guaranteed to be present? If not, what will happen here?

Reply:
@billsacks , at this point in the code, the user will have their cime-env loaded, so nf-config should be available. If not, NETCDF will be set to an empty string and therefore findNetcdf in cprnc/CMakeLists.txt will probably fail; I think this behavior is fine.

Reply:
Sounds good, thanks.